python/common/org/python/types/NotImplementedType.java

```diff
@@ -100,7 +108,8 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
     }
 
     @org.python.Method(
-        __doc__ = ""
+        __doc__ = "",
+        args = {"other"}
     )
     public org.python.Object __getitem__(org.python.Object other) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/freakboy3742/voc/file/2412364122/issues/source?bid=3235301&fileBranchId=3235629#l114)